CMakeLists.txt

```diff
@@ -127,12 +127,15 @@ set(lib_name ${PROJECT_NAME})
 
 set(srcs ${vtkIECTransformLogic_SRCS})
 add_library(${lib_name} SHARED ${srcs}) #SHARED = THERE IS DLL FILE
-
+target_include_directories(${lib_name} PUBLIC
+  $<BUILD_INTERFACE:${include_dirs}>
+  $<INSTALL_INTERFACE:include/${lib_name}>  # <prefix>/include/mylib
+)
 target_link_libraries(${lib_name} PUBLIC ${vtkIECTransformLogic_LIBS})
 
-message("ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ")
-message(${CMAKE_BINARY_DIR})
-message(${CMAKE_BINARY_DIR})
+#~ message("ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ")
```

Comment:
Can you please remove these? Feel free to commit and push directly after you have reviewed the commit and see that it's OK. Thanks!

Reply:
Thanks, I removed those lines. Do you want me also to remove the comment on include_directories line? I can also squash the commits if  you think it's necessary. Or maybe you have the option when clicking on "Merge with squash" (I have no rights).

Reply:
Ah I didn't realize you have no push rights. I'll grant you that now.